### ISSUE TITLE
Please release version with pastel 0.8 dependency

### ISSUE THREAD

**Issue by tas50** [piotrmurach/tty-table#34]
### Describe the problem

There's some gem conflicts accross the various TTY gems that make it really hard to consume the latest (ruby 2.7 compatible) releases of all the gems. Most things depend on pastel 0.8 now, but this gem is still hard pinned to the 0.7 series.

### Steps to reproduce the problem
Use this gem along with the latest version of `tty-box` or `tty-prompt`

### Actual behaviour
Conflicts such as this:

```
Bundler could not find compatible versions for gem "pastel":
  In Gemfile:
    chef was resolved to 16.5.53, which depends on
      tty-prompt (~> 0.22) was resolved to 0.22.0, which depends on
        pastel (~> 0.8)

    chef was resolved to 16.5.53, which depends on
      tty-table (~> 0.11.0) was resolved to 0.11.0, which depends on
        pastel (~> 0.7.2)
```

### Expected behaviour
Happy bundle updates

### Describe your environment

* OS version: macOS
* Ruby version: 2.7
* TTY::Table version: 0.11.0

**Comment by jmgarnier** [comment]
FYI It is also blocking the install of https://rubyjard.org/ the new debugger
```
Bundler could not find compatible versions for gem "tty-screen":
  In snapshot (Gemfile.lock):
    tty-screen (= 0.7.1)

  In Gemfile:
    ruby_jard was resolved to 0.3.0, which depends on
      tty-screen (~> 0.8.1)

    tty-table was resolved to 0.11.0, which depends on
      tty-screen (~> 0.7)
```

Thanks for a great tool by the way ! 🙇

**Comment by piotrmurach** [comment]
Hi Tim 👋 

Thanks for using the `tty-table`. I released `v0.12.0` which is compatible with the latest releases of `tty-box` and `tty-prompt`.

As an aside, I'm nearly done updating all tty gems(26) which I have been working on since the start of this year. Among many things, they are much smaller size-wise as I dropped few file artefacts like test files etc. The `tty-editor`, `tty-progressbar` and `tty-command` are still remaining and I hope to release them shortly.
